qutebrowser/browser/downloads.py

```diff
@@ -604,8 +619,7 @@ def get(self, url, page=None, fileobj=None, filename=None,
                          ui -> remove-finished-downloads is set to false.
 
         Return:
-            If the download could start immediately, (fileobj/filename given),
-            the created DownloadItem.
+            If the url is valid, the created DownloadItem.
```

Comment:
I think this change is no longer valid, as `get` calls `get_request` which only returns the `DownloadItem` if the download could be started immediately.